pkg/controller/cloneset/sync/cloneset_scale.go

```diff
@@ -60,6 +65,19 @@ func (r *realControl) Scale(
 		return false, nil
 	}
 
+	// If cloneset doesn't want to reuse pvc, clean up
+	// the existing pvc first. Then it looks like the pod
+	// is deleted by controller, new pod can be created.
+	if updateCS.Spec.ScaleStrategy.DisablePVCReuse {
+		uselessPVCs := getUselessPVCs(pods, pvcs)
```

Comment:
Should remove uselessPVCs in pvcs? then, pvcs only contains used pvcs.

Reply:
OK, I will fix it.